app/assets/javascripts/components/school_search.js.erb

```diff
@@ -83,14 +83,15 @@ document.addEventListener("DOMContentLoaded", function() {
         var school = findSchool(id);
 
         var suggestion =
-          '<label class="govuk-label govuk-label--s">' +
-          school.name +
-          "</label>" +
-          '<span class="govuk-hint">' +
-          school.address +
-          "</span>";
-
-        if (school.closeDate) suggestion += '<span class="govuk-hint">' + "Closed on " + school.closeDate + "</span>";
+          '<div class="flex"><div class="flex-1"><label class="govuk-label govuk-label--s">' + school.name + "</label>";
+        suggestion += '<span class="govuk-hint">' + school.address + "</span></div>";
+
+        if (school.closeDate)
+          suggestion +=
```

Comment:
Let's wrap this in curly braces, so it's clear it's a continuation of that line?